src/modules/charge-versions-upload/lib/mapper/adjustmentsMapper.js

```diff
@@ -11,16 +17,10 @@ const getAdjustments = data => {
     aggregate: parseFactor(data.chargeReferenceDetailsAggregateFactor)
   }
 
-  // Return Empty adjustments if there are no adjustments
-
-  return isEqual(adjustments, {
-    s126: null,
-    s127: false,
-    s130: false,
-    charge: null,
-    winter: false,
-    aggregate: null
-  })
+  const checkObjectProperties = (obj) => {
+    return Object.keys(obj).every((key) => isFalseyOrNull(obj[key]))
+  }
+  return checkObjectProperties(adjustments)
     ? {}
     : adjustments
```

Comment:
We could just assign the value to a variable instead of creating a function since we only use this the once:
```suggestion
  const noAdjustmentsToReturn = Object.keys(adjustments).every((key) => isFalseyOrNull(adjustments[key]))
  return noAdjustmentsToReturn
    ? {}
    : adjustments
```